sklego/preprocessing/dictmapper.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+from functools import partial
+
+
+class DictMapper(TransformerMixin, BaseEstimator):
+    """
+    Map the values of values of columns according to the input dictionary,
+    fall back to the default if the key is not present in the dictionary.
+
+    :param mapper: The dictionary containing the mapping of the values
+    :param default: The value to fall back to if the value is not in the mapper
+    """
+
+    def __init__(self, mapper, default):
+        self.mapper = mapper
+        self.default = default
+
+    def fit(self, X, y=None):
```

Comment:
nitpick, but could you add a docstring here? feel free to copy it from another class.